lightkurve/search.py

```diff
@@ -63,24 +64,24 @@ def _add_columns(self):
             prefix = {'Kepler': 'Quarter', 'K2': 'Campaign', 'TESS': 'Sector'}
             for idx in range(len(self.table)):
                 self.table['#'][idx] = idx
-                mission = self.table['obs_collection'][idx]
+                mission = self.table['project'][idx]
                 seqno = self.table['sequence_number'][idx]
                 if mission == 'Kepler' and self.table['sequence_number'].mask[idx]:
                     seqno = re.findall(r".*Q(\d+)", self.table['description'][idx])[0]
                 self.table['observation'][idx] = "{} {} {}".format(mission,
-                                                                   prefix[mission],
+                                                                   prefix.get(mission, ""),
                                                                    seqno)
         except Exception:
             # be tolerant of any MAST API changes
             # which may cause the code above to fail
-            log.warning("Unexpected data encountered in the ``SearchResult`` "
-                        "constructor; the MAST API may have changed.")
+            log.debug("Unexpected data encountered in the ``SearchResult`` "
+                      "constructor; the MAST API may have changed.")
 
     def __repr__(self, html=False):
         out = 'SearchResult containing {} data products.'.format(len(self.table))
         if len(self.table) == 0:
             return out
-        columns = ['#', 'observation', 'target_name', 'productFilename', 'distance']
+        columns = ['#', 'observation', 'target_name', 'productFilename', 'provenance_name', 'distance']
```

Comment:
provenance_name is quite ambiguous, it's possible to rename these columns if we wanted to. Should we rename it something a little more user friendly? The advantage to leaving it as is would be that it's consistent with MAST.

Reply:
Resolved: I renamed `provenance_name` to `author`.